.github/workflows/gradle.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
```

Comment:
I think we have newer version of this, checkout releases, there should be migration.zip with improved flows

Reply:
> These dependencies are required to compile:
> 
> ```groovy
> dependencies {
>     compile("com.github.GTNewHorizons:GT5-Unofficial:experimental-SNAPSHOT:dev") { setChanging(true) }
>     compile "cpw.mods:ironchest:1.7.10-6.0.62.742:deobf"
>     compile "codechicken:NotEnoughItems:1.7.10-1.0.5.120:dev"
>     compile "player-api:PlayerAPI:1.7.10:1.4"
>     compile "render-player-api:RenderPlayerAPI:1.7.10:1.4"
>     compile "smart-render:SmartRender:1.7.10:2.1"
>     compile "tconstruct:TConstruct:1.7.10-1.8.8.build1003:deobf"
> }
> ```

You can set Player-API, RenderPlayerAPI and SmartRender to compileOnly.